src/setup.py

```diff
@@ -18,7 +18,7 @@
         'environment_tools>=1.1.0,<1.2.0',
         'kazoo>=2.2.0',
         'PyYAML>=3.11',
-        'paasta-tools==0.81.26',
+        'paasta-tools==0.83.2',
```

Comment:
I know that it's too much for just bumping up the `paasta-tools` version, but can we use this opportunity and get rid of this duplication the same way as it's done in https://github.com/Yelp/paasta/blob/master/setup.py#L24 ?